test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1485,24 +1485,25 @@ def listdir(self, hostname=None, path=None, sudo=False, runas=None,
         :param level: Logging level.
         :type level: int
         :returns: A list containing the names of the entries in
-                  the directory
+                  the directory or an empty list in case no files exist
         """
+        retvalerr = []
 
         if path is None:
-            return None
```

Comment:
Is it really easier to define and return retvalerr, rather then just returning an empty list?

Reply:
Not really .. I guess I was just thinking that if we ever change it back to None or something else then you'd only have to change it in one place

Reply:
Makes sense, just curious